src/game_api/script/usertypes/entities_chars_lua.cpp

```diff
@@ -34,6 +34,23 @@ void register_usertypes(sol::state& lua)
         &Inventory::money,
         "bombs",
         &Inventory::bombs,
+        "time_of_death",
+        &Inventory::time_of_death,
+        "player_slot",
+        sol::property([](Inventory& i) -> int8_t
```

Comment:
Docs don't handle this well, right? Place in a local variable and move in here to make the docs look nicer.